andela/bad_ass-ah-frontend
# #164489887  Login via social media

**What does this PR do?**

- This PR will allow users to login via social Media 

**Description of task to be completed**

- A given user will be able to login via their google account

- A given user will be able to login via their facebook account

- A given user will be able to login via their twitter account

**How should this be manually tested?**

- Make sure the server is running on the server is running or run it with the command `npm run dev`

- Go to the following url `localhost:3000/login`

- The click respectively to the button `google`, `facebook`, `twitter` to login

**What are the relevant pivotal tracker stories??**

- [#164489887 ](https://www.pivotaltracker.com/story/show/164489888)

- **Screen shots (if appropriate)**
![image](https://user-images.githubusercontent.com/46047244/58490067-c26ec080-816c-11e9-9ae3-fbac3b052674.png)

![image](https://user-images.githubusercontent.com/46047244/58490143-e3cfac80-816c-11e9-9c19-619e8adf33db.png)
